src/Whoops/Run.php

```diff
@@ -288,6 +293,31 @@ public function sendHttpCode($code = null)
         return $this->sendHttpCode = $code;
     }
 
+    /**
+     * Should Whoops exit with a specific code on the CLI if possible?
+     * Whoops will exit with 1 by default, but you can specify something else.
+     *
+     * @param int $code
+     *
+     * @return int
+     *
+     * @throws InvalidArgumentException
+     */
+    public function sendExitCode($code = null)
+    {
+        if (func_num_args() == 0) {
+            return $this->sendExitCode;
+        }
+
+        if ($code < 0 || 255 <= $code) {
+            throw new InvalidArgumentException(
+                "Invalid status code '$code', must be between 0 and 254"
```

Comment:
To match the conditional above:

```suggestion
                "Invalid status code '$code', must be between 1 and 255"
```

Reply:
I’ve misread the conditional, I’ve read it as `!(0 < $code && $code <= 255)`. Yeah, your reasoning is stellar, thanks for the patient answer!